tenablesc.json

```diff
@@ -1754,37 +1791,131 @@
                 }
             ],
             "versions": "EQ(*)"
+        },
+        {
+            "action": "list credentials",
+            "description": "Lists the credentials available in Tenable.sc",
```

Comment:
Does this include the credential passwords?

Reply:
Hi @alexa-phantom 
No, this action does not contain the credential's password, This action contains only "Credential ID", "Credential Name", "Description" & "Type". I have attached 1 action run for the same.

<img width="1394" alt="Screenshot 2022-09-07 at 10 40 45 AM" src="https://user-images.githubusercontent.com/72184535/188793600-e8a2b2f2-5020-4b78-9ce3-fe36113f4af9.png">

Reply:
Thanks for the clarification. What are your thoughts on pointing out that no password will be shown?

Reply:
Yes, Sure @alexa-phantom 
We are mentioning that point in the action verbose.

<img width="767" alt="Screenshot 2022-09-08 at 12 22 33 PM" src="https://user-images.githubusercontent.com/72184535/189054632-c9576876-d3ec-4cde-86f1-25c346442cfe.png">